web/www/horas/Latin/Sancti/11-23.txt

```diff
@@ -1,96 +1,4 @@
-[Rank]
-S. Clementis Papae et Martyris;;Duplex;;3;;vide C2
-
-[Rank1570]
-S. Clementis Papae et Martyris;;Semiduplex;;2;;vide C2
-
-[RankNewcal]
-S. Clementis Papae et Martyris;;Duplex optional;;2;;vide C2
-
-[Rule]
-vide C2;
-9 lectiones
-OPapaM=Clem�ntem;
-
-[Ant Vespera]
-Orante * sancto Clemente, apparuit ei Agnus Dei.;;109
```

Comment:
The asterisk has moved as a result of pulling this in from 11-23t.  My breviary gives "Orante * sancto..." as the correct placement.